_assets/scss/design-guide.scss

```diff
@@ -22,72 +22,135 @@ table {
   margin-bottom: $base-spacing * 2;
 }
 
-// Custom styling for the inverted feedback button in the header.
-// This will likely be removed/replaced by a global site search.
-// The @extend .inverted is not working in:
-// https://github.com/AusDTO/gov-au-ui-kit/blob/master/assets/sass/components/_page-header.scss#L288
 
-header[role='banner'] {
-  .page-header {
-    padding-top: 0;
-  }
+.hero {
+  padding-top: 0;
 
-  .page-header__logo {
-    padding-top: 0;
-    margin-bottom: $tiny-spacing;
+  .wrapper {
+    @include ie-lte(8) {
+      @include ie-clearfix;
+    }
+
+    background-image: url('/assets/hero.svg');
```

Comment:
Can we inline the SVG to save that precious http request (prior to http/2 rollout)?
E.G.:
```css
url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='1000' height='20' version='1.1'><polyline fill='white' points='0,0 1000,0 0,20'/></svg>") left top / 100% auto no-repeat
```

http://meyerweb.com/eric/tools/dencoder/